### PR TITLE
CSV splitter

### PR DESCRIPTION
**Pull request by eanders**
This script lets you split a large set of HMIS CSVs by a set of ProjectIDs.

### CODE REVIEW

**Comment by eanders** [comment]
@gigxz this was working, but I accepted a rubocop fixup that broke it.  It's close, but I'll have to finish up a bit later.

File: app/models/grda_warehouse/tasks/hmis_csv_splitter.rb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    end
+
+    private def project_related
+      [
+        'Project.csv',
+        'Inventory.csv',
+        'ProjectCoC.csv',
+        'Affiliation.csv',
+        'Funder.csv',
+      ]
```
Comment (gigxz) [inline app/models/grda_warehouse/tasks/hmis_csv_splitter.rb:144]: Should the 2024 file names be added here? Ce participation and hmis participation

Comment (gigxz) [reply app/models/grda_warehouse/tasks/hmis_csv_splitter.rb:144]: ProjectCoC too

Comment (eanders) [reply app/models/grda_warehouse/tasks/hmis_csv_splitter.rb:144]: At the moment, this is 2022 only, I figured we'd update that after 10/1 or when we need to split a 2024 file.  That may be the only change necessary, but until we receive a 2024 file it felt premature.

Comment (gigxz) [reply app/models/grda_warehouse/tasks/hmis_csv_splitter.rb:144]: Ok. Yep only needed for 2022 for now.

**Review by gigxz — APPROVED** [review]

**Pull request merged**